_opcodes/op084-iwd2.html

```diff
@@ -1,17 +1,13 @@
 ---
 n: 84
-name: "Stat: Magical Fire Resistance Modifier"
-param1: "Statistic Modifier"
-param2: "Type"
+name: "Empty:"
+param1: "Irrelevant"
+param2: "Irrelevant"
 bg1: 0
 bg2: 0
 bgee: 0
 iwd1: 0
 iwd2: 1
 pst: 0
 ---
-*DO NOT USE*
-<br/>Applies the modifier value specified by the 'Statistic Modifier' field in the style specified by the 'Type' field.
-<br/>Known values for 'Type' are:
-<br/>0   Cumulative Modifier -> Damage = Damage + 'Statistic Modifier' value
-<br/>1   Flat Value Modifier -> Damage = 'Statistic Modifier' value
+This opcode has no effect.
```

Comment:
I don't know this yet. Will revert the changes for all *do not use* iwd2 opcodes